deploy.sh

```diff
@@ -47,7 +47,6 @@ echo "${BOLD}Deploying API...${NONE}"
 ./gradlew :api:appengineDeploy
 
 echo "${BOLD}Deploying UI...${NONE}"
-./gradlew :ui:appengineDeploy
-
-
-
+cd ui
+ng build  # Change this to `ng build --environment=prod` to minify.
+gcloud app deploy
```

Comment:
The only reason I'm OK with this is we have set the project and account above in the script. But if we can specify them on the command line here instead, fine with me.